ddtrace/contrib/psycopg/patch.py

```diff
@@ -69,7 +79,13 @@ def __init__(self, conn, pin=None, cursor_cls=None):
         if not cursor_cls:
             # Do not trace `fetch*` methods by default
             cursor_cls = Psycopg2TracedCursor
-            if config.dbapi2.trace_fetch_methods:
+            if config.psycopg.trace_fetch_methods or config.dbapi2.trace_fetch_methods:
```

Comment:
@majorgreys @brettlangdon what are the odds someone would set `config.psycopg.trace_fetch_methods` to something other than `True`? If this is explicitly set to `False`, and `config.dbapi2.trace_fetch_methods` is set to `True`, then we would have a bug.

Reply:
something other than `True` is probably fairly low, but "truthy" check here should be good enough.

> If this is explicitly set to False, and config.dbapi2.trace_fetch_methods is set to True, then we would have a bug

you mean, if `config.psycopg.trace_fetch_methods = False` and `config.dbapi2.trace_fetch_methods = True`?

Reply:
Right, the default for these being `False` makes deciding here difficult. Someone explicitly disabling `config.psycopg.trace_fetch_methods` but still enabling via `config.psycopg.trace_fetch_methods` would find fetch methods traced still. Upon examining the logs they'd come across the deprecation warning and hopefully have already seen the release notes for using `config.psycopg.trace_fetch_methods`, in which case they'd have a clear resolution (not setting `config.psycopg.trace_fetch_methods`). The present behavior then doesn't seem to be a bug but an inconvenience that's coming from the deprecation process.